packages/babel-traverse/src/path/family.ts

```diff
@@ -213,14 +213,14 @@ export function _getKey<T extends t.Node>(
 
   if (Array.isArray(container)) {
     // requested a container so give them all the paths
-    return container.map((_, i) => {
+    return container.map((v, i) => {
       return NodePath.get({
         listKey: key,
         parentPath: this,
         parent: node,
         container: container,
         key: i,
-      }).setContext(context);
+      }).setContext(v.context);
```

Comment:
I think `v.context` should be supplied as a fallback of `context`.

Reply:
Here the context defaults to `true`, so if no specific context is passed it will be used the one of the current `NodePath` (It may not be the right one).
Because of this we can't fallback since context is always defined when calling `#get()`.
https://github.com/babel/babel/blob/468a43c8bbc9cccfc98a25fee17de9347827ade1/packages/babel-traverse/src/path/family.ts#L188-L202

Reply:
In this case we can use `v.context` when `context` is `true`, otherwise when a `context` is indeed passed as `TraversalContext`, we should respect the input, though.